lib/html.rb

```diff
@@ -35,7 +35,7 @@ def self.generate(description)
     File.write(File.join(target, "index.html"), template.render(binding))
     File.write(File.join(target, "assets/description.js"),<<-EOT
       function getDescription() {
-        return JSON.parse('#{description.to_hash.to_json}')
+        return JSON.parse('#{description.to_hash.to_json.gsub("\\\"", "&quote").gsub("'", "&#39;")}')
```

Comment:
Line is too long. [101/100]<br>Prefer single-quoted strings inside interpolations.